### PR TITLE
B feature/maven profiles

### PR DESCRIPTION
**Pull request by mariobasic**

### CODE REVIEW

**Comment by stanislavgrujic** [comment]
Did you communicate this change with the PM so that the build system is properly updated?

**Comment by mariobasic** [comment]
Yes I did, but just in case it's not fully understood don't merge PR yet, I will go through changes with him once again to make sure and give you definite response.

**Comment by mariobasic** [comment]
I talked with PM, he made suggestion to make `mvn clean install` build `frontend` by default, and to make specialized profile which does not build `frontend` and left final decision to you. 

Should I make change to build `frontend` with default profile, or leave it the way it is?

**Comment by stanislavgrujic** [comment]
Do as agreed with the PM, since he is also involved in the build process.

File: pom.xml [hunk 1]
```diff
@@ -200,52 +200,77 @@
...
+        <profile>
+            <id>dev</id>
+        </profile>
+        <!-- skip unit test -->
+        <profile>
+            <id>skip-tests</id>
+            <properties>
+                <maven.test.skip>true</maven.test.skip>
+            </properties>
+        </profile>
```
Comment (stanislavgrujic) [inline pom.xml]: Why do you need this profile? You can always omit tests by passing `-DskipTests` argument to `mvn` call.

**Pull request merged**